modules/power/default.nix

```
@@ -0,0 +1,47 @@
+{ config, lib, ... }:
+
+with lib;
+
+let
+  cfg = config.power;
+
+  onOff = cond: if cond then "on" else "off";
+in
+
+{
+  options = {
+    power.restartAfterPowerFailure = mkOption {
+      type = types.nullOr types.bool;
+      default = null;
+      description = lib.mdDoc ''
```

```suggestion
      description = ''
```

All the `lib.mdDoc` calls are deprecated and need to be removed